Makefile

```diff
@@ -351,4 +351,16 @@ test:
 	skipper make $(SKIPPER_PARAMS) _test
 
 _test: $(REPORTS)
+	if [ -d /tmp/assisted_test_infra_logs ]; then rm -rf /tmp/assisted_test_infra_logs; fi
```

Comment:
No need for this. Done here:
https://github.com/openshift/assisted-test-infra/blob/master/discovery-infra/tests/conftest.py#L81